charts/argo-cd/templates/_common.tpl

```diff
@@ -46,6 +46,7 @@ helm.sh/chart: {{ include "argo-cd.chart" .context }}
 {{ include "argo-cd.selectorLabels" (dict "context" .context "component" .component "name" .name) }}
 app.kubernetes.io/managed-by: {{ .context.Release.Service }}
 app.kubernetes.io/part-of: argocd
+app.kubernetes.io/version: {{ .Chart.AppVersion }}
```

Comment:
https://helm.sh/docs/chart_best_practices/labels/#standard-labels

<img width="1329" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/349d0f9e-ca03-4cbe-90a2-f9174f5c39e4">

Reply:
`argo-cd.labels` is used almost all of all resources, so I think we set it here. 🤔